src/Compute/Compute/help/Set-AzVmssDiskEncryptionExtension.md

```diff
@@ -36,10 +36,25 @@ $VaultName= "MyKeyVault"
 $KeyVault = Get-AzKeyVault -VaultName $VaultName -ResourceGroupName $RGName
 $DiskEncryptionKeyVaultUrl = $KeyVault.VaultUri
 $KeyVaultResourceId = $KeyVault.ResourceId
-PS C:\> Set-AzVmssDiskEncryptionExtension -ResourceGroupName $RGName -VMScaleSetName $VmssName -DiskEncryptionKeyVaultUrl $DiskEncryptionKeyVaultUrl -DiskEncryptionKeyVaultId $KeyVaultResourceId
+
+PS C:\> Set-AzVmssDiskEncryptionExtension -ResourceGroupName $RGName -VMScaleSetName $VmssName -DiskEncryptionKeyVaultUrl $DiskEncryptionKeyVaultUrl -DiskEncryptionKeyVaultId $KeyVaultResourceId```
+
+This command enables encryption on all disks of all Windows VMs in a VM scale set.
+
+### Example 2
 ```
+$RGName = "MyResourceGroup"
+$VmssName = "MyTestVmss"
+$VaultName= "MyKeyVault"
+$KeyVault = Get-AzKeyVault -VaultName $VaultName -ResourceGroupName $RGName
+$DiskEncryptionKeyVaultUrl = $KeyVault.VaultUri
+$KeyVaultResourceId = $KeyVault.ResourceId
+$VolumeType = "Data"
+
+PS C:\> Set-AzVmssDiskEncryptionExtension -ResourceGroupName $RGName -VMScaleSetName $VmssName -DiskEncryptionKeyVaultUrl $DiskEncryptionKeyVaultUrl -DiskEncryptionKeyVaultId $KeyVaultResourceId
+ -VolumeType $volumeType```
 
-This command enables encryption on all disks of all VMs in the VM scale set.
+This command enables encryption on all disks of all Lunix VMs in a VM scale set.
```

Comment:
typo: Lunix -> Linux

Reply:
For Linux data disk encryption on VMSS - I think we may need to add a note here that the data disks must be formatted and mounted and prepared properly within the OS using a persistent device identifier before this command is run or they will not be encrypted.    Some of this is described here, although a separate how to page just for data disk preparation might be helpful to add in the future:  https://docs.microsoft.com/en-us/azure/security/azure-security-disk-encryption-prerequisites#bkmk_LinuxPrereq